travis/test.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+if [ "$ARCH" == "" ]; then
+    echo 'Error: $ARCH is not set'
+    exit 1
+fi
+
+if [ "$ARCH" != "x86_64" ]; then
+    echo 'NOTICE: $ARCH is not supported yet'
+    exit 0
+fi
+
+TARGET=$1
+if [ "$TARGET" == "" ]; then
+    echo 'Error: $TARGET is not set'
+    exit 1
+fi
+
+
+# use RAM disk if possible
+if [ "$CI" == "" ] && [ -d /dev/shm ]; then
+    TEMP_BASE=/dev/shm
+else
+    TEMP_BASE=/tmp
+fi
+
+BUILD_DIR=$(mktemp -d -p "$TEMP_BASE" linuxdeploy-plugin-qt-build-XXXXXX)
+
+cleanup () {
+    if [ -d "$BUILD_DIR" ]; then
+        rm -rf "$BUILD_DIR"
+    fi
+}
+
+trap cleanup EXIT
+
+
+wget -N https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-"$ARCH".AppImage
+export LINUXDEPLOY_BIN=${PWD}/linuxdeploy-"$ARCH".AppImage
+chmod +x ${LINUXDEPLOY_BIN}
+
+pushd "$BUILD_DIR"
+
+git clone --depth=1 https://github.com/linuxdeploy/linuxdeploy-plugin-qt-examples.git
+
+## Install additional build stuff
+sudo apt-get install -y software-properties-common
+sudo add-apt-repository -y 'ppa:beineri/opt-qt-5.10.1-trusty'
+sudo apt update
+sudo apt install -y qt510-meta-minimal qt510declarative qt510webengine mesa-common-dev
+
+export QT_PREFIX=/opt/qt510
+mkdir -p $HOME/.config/qtchooser
+echo "${QT_PREFIX}/bin" > $HOME/.config/qtchooser/qt5.10.conf
+echo "${QT_PREFIX}/lib" >> $HOME/.config/qtchooser/qt5.10.conf
+
+$QT_PREFIX/bin/qt510-env.sh
```

Comment:
You don't need any of that `export` calls. You just don't call the env script correctly. You must call it through `source`.